pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests that a device has the right attributes, arguments and methods."""
+# pylint: disable=no-self-use
+import numpy as np
```

Comment:
Certain demonstrations use the updated version of NumPy provided by PennyLane, which causes arrays to be represented using the `pennylane.numpy.tensor` class.

My suggestion here would aim at making sure that edge cases are covered for many devices. Let me know, however, if some problems would arise by making this change here. :)
```suggestion
from pennylane import numpy as np
```
----

*Small example:*
```python
from pennylane import numpy as np

n_layers = 5
np.random.rand(n_layers, 1)
```
```
tensor([[0.20917477],
        [0.6499307 ],
        [0.87817394],
        [0.13879325],
        [0.09579755]], requires_grad=True)
```